enterprise/api/v1/views.py

```diff
@@ -219,6 +219,9 @@ def course_enrollments(self, request, pk):
     def enterprise_learners(self, request, pk):
         """
         Creates a set of enterprise_learners by enrolling them in the specified course.
+        If request.data contains 'email' field it's considered prioritized
```

Comment:
This method is now very long. Is it possible to split it into a few smaller methods?  I am worried about legibility.

Reply:
did a couple of refactors to clean it up